src/infrastructure/notification/notification.core-repository.ts

```diff
@@ -0,0 +1,33 @@
+import { Injectable } from '@nestjs/common';
+import { PrismaService } from 'src/infrastructure/prisma/prisma.service';
+import { NotificationRepository } from 'src/domain/notification/notification.repo';
+import { NotificationResponseEntity } from 'src/domain/notification/entity/notification.response.entity';
+import { NotificationsRequestDto } from 'src/interfaces/controller/notification/dto/notification.request.dto';
+
+@Injectable()
+export class NotificationCoreRepository implements NotificationRepository {
+  constructor(private readonly prisma: PrismaService) {}
+
+  async findMany(args: NotificationsRequestDto) {
+    let query = {};
+    if (!args.cursor) {
+      query = {
+        take: args.size,
+        orderBy: { notification_id: 'desc' },
+        where: { user_id: args.userId },
+      };
+    } else {
+      query = {
+        take: args.size,
+        orderBy: { notification_id: 'desc' },
+        where: { user_id: args.userId, notification_id: { lt: args.cursor } },
+      };
+    }
+    const rawData = await this.prisma.notification.findMany(query);
```

Comment:
```suggestion
    const queryResult = await this.prisma.notification.findMany(query);
```
rawData라는 네이밍이 조금 범위가 넓은 것 같습니다. rawData라고 하면 보통 bineryData를 생각하는데, 지금 받는 결과값은 쿼리에 대한 결과 값이니, 좀 더 거기에 의미를 두게 네이밍을 바꾸는게 좋지 않을까요?